.github/workflows/discord-release-announcement.yaml

```diff
@@ -0,0 +1,18 @@
+name: Discord Release Announcement
+
+on:
+  release:
+    types: [prereleased, published]
+
+jobs:
+  delimiter-test:
+    runs-on: ubuntu-latest
+    name: Send message to discord
+    steps:
+      - name: parse and send message
+        uses: michelengelen/discord-message-action@02af30a15955ecf718049bc33b0efabf6f626e0b
+        with:
+          webhook-url: ${{ secrets.DISCORD_WEBHOOK_URL }}
+          username: MUI Releases
+          avatar-url: 'https://i.ibb.co/YfMzb7r/logo.png'
```

Comment:
If we have one that would be great ... I got this URL from @joserodolfofreitas, since all assets we have are basically `svg`s

Reply:
I would much rather not see this line of code linked in https://www.notion.so/mui-org/Brand-usage-to-keep-in-sync-9cdf2e7410154ad4b4067c5a7e2c713a. The shorter the list, the simplest we can deploy brand changes. 

@danilo-leal I could see a ourselves adding our brand icons in a folder under static. It would match with https://www.notion.so/mui-org/Branding-book-42011f0b6ad44e7ba4e507c00b2da13a. We have this one today https://github.com/mui/material-ui/blob/master/docs/public/static/logo.png.

Reply:
Maybe it's best to use the GitHub link in the end, as that's surely the link that will keep being updated in the future!

Reply:
@danilo-leal I just noticed that the logo on github has some incorrect corner radii:

<img width="788" alt="Screenshot 2024-02-08 at 14 34 58" src="https://github.com/mui/mui-x/assets/32863416/1f91c31c-b99a-4fff-84fc-37c7811dab7f">

Reply:
Yup, aware! The revised version of the logo hasn't been rolled out yet — will get to it soon!

Reply:
>  I could see ourselves adding our brand icons in a folder under static.

This would be ideal;